### PR TITLE
Test/combine original files with mine

### PR DESCRIPTION
**Pull request by nkuik**
This change is [Reviewable](https://reviewable.io/reviews/voku/dotfiles/35)

### CODE REVIEW

File: .bash_profile [hunk 1]
```diff
@@ -1,16 +1,40 @@
...
+    . /etc/bash_completion
+fi
+
+
+function_exists() {
+    declare -f -F $1 > /dev/null
+    return $?
+}
+
+alias stt="C:/Program\ Files/Sublime\ Text\ 3/sublime_text.exe"
```
Comment (voku) [inline .bash_profile]: move to ".aliases" ?

File: .bash_profile [hunk 2]
```diff
@@ -69,13 +97,48 @@ shopt -s no_empty_cmd_completion;
...
+# Example format: plugins=(rails git textmate ruby
+# lighthouse)
+# Add wisely, as too many plugins slow down shell
+# startup.
+plugins=$(echo $CONFIG_BASH_PLUGINS | sed 's/(//g' | sed 's/)//g')
 
   source $HOME/.redpill/redpill-init-bash.sh
 fi
+
+function hdrive {
```
Comment (voku) [inline .bash_profile]: move to ".functions" ?

File: .bash_profile [hunk 3]
```diff
@@ -69,13 +97,48 @@ shopt -s no_empty_cmd_completion;
...
+     cd /mnt/h/code
+}
+
+function local_code {
+    cd /mnt/c/Users/naku0510/local_code
+}
+
+alias stt="C:/Program\ Files/Sublime\ Text\ 3/sublime_text.exe"
```
Comment (voku) [inline .bash_profile]: duplicate code?

File: .bash_profile [hunk 4]
```diff
@@ -69,13 +97,48 @@ shopt -s no_empty_cmd_completion;
...
+
+HISTSIZE=5000
+HISTFILESIZE=10000
```
Comment (voku) [inline .bash_profile]: merge / move to ".exports"

File: .bashrc [hunk 5]
```diff
@@ -1,54 +1,14 @@
...
+    return 0
 fi
 
 # try to load global-bashrc
 if [ -f /etc/bashrc ]; then
-  . /etc/bashrc
+      . /etc/bashrc
 fi
 
+source /usr/share/bash-completion/completions/git
```
Comment (voku) [inline .bashrc]: check if source exists?

File: .bash_history [hunk 6]
```diff
@@ -0,0 +1,1637 @@
+git clone git@github.com:UiPath/ReFrameWork.git
+cd ..
```
Comment (voku) [inline .bash_history]: Please do not commit this file! ... maybe we need a .gitignore entry for that?

**Comment by nkuik** [comment]
Sorry. Last time this will happen.

**Pull request closed without merging**